taxopy/core.py

```diff
@@ -253,6 +266,8 @@ class Taxon:
         The NCBI taxonomic identifier the object represents (e.g., 9606).
     name: str
         The name of the taxon (e.g., 'Homo sapiens').
+    names: list
+        All names of the taxon as a list of kind and name (e.g., `[('authority', 'Homo sapiens Linnaeus, 1758'), ('scientific name', 'Homo sapiens'), ('genbank common name', 'human')])`.
```

Comment:
I'm using a line length of ~90 characters for the docstrings, consistent with the code formatting